implementing-domain-logic/event-handling/dispatching-events.md

```diff
@@ -1,6 +1,6 @@
 # Dispatching Events
 
-Event publication from a couple of locations when your Axon Framework application.
+Event publication can from a couple of locations within your Axon Framework application.
```

Comment:
I think we need to add something like `occur` or `happen` after the `can`.

Reply:
Yeah. I'll use `occur`